data/json/items/fake.json

```diff
@@ -28,8 +28,22 @@
     "id": "toolset",
     "copy-from": "fake_item",
     "type": "TOOL",
-    "name": { "str": "integrated welder" },
-    "flags": [ "TRADER_AVOID", "FIRESTARTER", "USES_BIONIC_POWER" ]
+    "name": { "str": "integrated multitool" },
+    "flags": [ "TRADER_AVOID", "FIRESTARTER", "USES_BIONIC_POWER" ],
+    "use_action": [ "HAMMER", "CROWBAR" ],
+    "qualities": [
+      [ "SAW_W", 1 ],
```

Comment:
```suggestion

```

Reply:
I left saw_w because the Multitool item itself has this property.

Reply:
I think it was fine as a welder only, but there's no wood saw in here.  Especially because I think there is more sense in having a giant arm replacement saw bionic if people really want wood sawing cbms.

Reply:
wood sawing 1 is Swiss Army Knife, Multitool.
wood sawing 2 is for a proper bow saw or wood saw. 
my thinking anyhow.

Reply:
We don't have swiss army knife tools, so no.

Reply:
> We don't have swiss army knife tools, so no.

What is [this](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/items/melee/swords_and_blades.json#L240) then? Also [this](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/items/melee/swords_and_blades.json#L187) and of course [this](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/items/tool/workshop.json#L675).

Reply:
After reminding myself what SAW_W: 1 implies, yea this can stay as it only enables very small scale wood shaping jobs.  I had forgotten that we finally did sort out a breakpoint for a tiny almost toy wood saw as opposed to a fully capable wood saw.